content/TruenasApps/StableApps/PlexApp.md

```diff
@@ -0,0 +1,263 @@
+---
+title: "Plex"
+description: "Provides installation instructions to configure and deploy the Plex app in TrueNAS."
+weight:
+aliases:
+tags:
+- apps
+- media
+keywords:
+- nas data storage
+- software storage solutions
+---
+
+
+Plex is a media server that allows you to stream your media (music, movies, live TV, etc.) to any Plex client.
+
+{{< include file="/static/includes/AppsUnversioned.md" >}}
+
+## Before You Begin
+Before you install the Plex app:
+
+* Set up a Plex account.
+
+  After installing the Plex app and logging into Plex through the link in TrueNAS, if you have not already configured your Plex account media server, Plex shows the configuration screens to set up the media server, add libraries, and customize your Plext account.
+
+{{< include file="/static/includes/apps/AppsStableBeforeYouBegin.md" >}}
+
+* Go to **Datasets**, and select the pool or dataset where you want to add the Plex datasets.
+  For example, */tank/apps/plex* or */tank/plex*.
+  You can use either an existing pool or [create a new one]({{< relref "CreatePoolWizard.md" >}})
+
+  Create the two datasets Plex uses for storage volumes:
+  * **data** for user media data storage
+  * **config** for Plex application configuration storage.
+
+  You can create a dataset for log data or use a temporary directory option for log data.
+  Transcode data is not useful or meant for persistent storage, so using a temporary directory option is a better option.
+
+  When creating the above datasets select the **apps** dataset preset.
+
+  You can set up the ACLs for these datasets after adding them or use the app installation wizard to configure the ACLs and ACL entries.
```

Comment:
```suggestion
  You can set up the permissions (ACLs) for these datasets after adding them or use the app installation wizard to update the existing ACLs.
```

Reply:
This is not really correct, you can add the permissions in the wizard. I'll adjust this sentence for better clarity.